src/BloomBrowserUI/bookEdit/js/bloomEditing.js

```diff
@@ -440,22 +440,23 @@ function MakeSourceTextDivForGroup(group) {
   else {
     $(divForBubble).remove();//no tabs, so hide the bubble
     return;
-  }
+    }
+
+    var showEvents = false;
+    var hideEvents = false;
+    var shouldShowAlways = true;
+
+  
+    if(checkMightCauseHorizontallyOverlappingBubbles(group)) {
```

Comment:
There's another cluster of lines around 935 where a (local) variable called onFocusOnly is determined from bloom-showOnlyWhenTargetHasFocus. I haven't studied in enough depth to know there's a relationship, I just wondered.
(At least I'm guessing that's what I meant; it's been too long to remember for sure...)